src/commands/image/imageSource/buildImageInAzure/SourcePathStep.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from '@microsoft/vscode-azext-utils';
+import * as path from 'path';
+import { browseItem } from '../../../../constants';
+import { localize } from '../../../../utils/localize';
+import { type BuildImageInAzureImageSourceContext } from './BuildImageInAzureImageSourceContext';
+
+export class SourcePathStep extends AzureWizardPromptStep<BuildImageInAzureImageSourceContext> {
+    public async prompt(context: BuildImageInAzureImageSourceContext): Promise<void> {
+        await context.ui.showQuickPick([browseItem], {
+            placeHolder: localize('sourceDirectoryPick', 'Choose your source code directory')
+        });
+
+        context.srcPath = (await context.ui.showOpenDialog({
```

Comment:
That's fair, my only question is, if we end up suggesting the wrong location using the Dockerfile as reference, do you think most people will think it intuitive enough to hit browse to pick the correct location?

Reply:
Oh I see you also mentioned showing "." first, let's maybe talking offline so I can fully understand which places could make sense to show

Reply:
I think users would understand it. 

I think it's similar to Static Web Apps, but basically, it shows your root folder and every sub-directory as a quickpick with browse at the bottom.

I don't think it would be anymore confusing that just asking where the source folder is, at least. It's also an additional click which may be annoying (especially if the src folder is one that we could list for users from the get-go).